pygfx/objects/_base.py

```diff
@@ -389,152 +379,52 @@ def iter(self, filter_fn=None, skip_invisible=False):
         elif filter_fn(self):
             yield self
 
-        for child in self._children:
+        for child in self.children:
             yield from child.iter(filter_fn, skip_invisible)
 
-    def update_matrix(self):
-        p, r, s = self.position, self.rotation, self.scale
-        hash = p.x, p.y, p.z, r.x, r.y, r.z, r.w, s.x, s.y, s.z
-        if hash != self._transform_hash:
-            self._transform_hash = hash
-            self._matrix.compose(self.position, self.rotation, self.scale)
-            self._matrix_world_dirty = True
-
     @property
-    def matrix(self):
-        """The (settable) transformation matrix."""
-        return self._matrix
+    def transform_sequence(self) -> ChainedTransform:
+        own_sequence = ChainedTransform([self.transform])
 
-    @matrix.setter
-    def matrix(self, matrix):
-        self._matrix.copy(matrix)
-        self._matrix.decompose(self.position, self.rotation, self.scale)
-        self._matrix_world_dirty = True
+        if self.parent is None:
+            return own_sequence
+        else:
+            return self.parent.transform_sequence @ own_sequence
 
     @property
-    def matrix_world(self):
-        """The world matrix (local matrix composed with any parent matrices)."""
-        return self._matrix_world
+    def bounding_box(self):
```

Comment:
I'm a  bit wary of computed properties that potentially take a long time to compute; a property raises the expectation that the result is (nearly) readily available, which is not true if this would be a scene with 10k children :) So my preference would be to use `get_bounding_box()`.

Reply:
Sure, we can make it normal functions again. I never had the impression that computed properties give any performance guarantees. 

Alternatively, if it does become too slow, we could also add a cache for it.